components/system-api/src/private.rs

```diff
@@ -16,6 +16,188 @@
 //!
 //! Private API includes requests that are available only to the blockchain
 //! administrators, e.g. shutting down the node.
+//!
+//! # Table of Contents
+//!
+//! - [List peers](#list-peers)
+//! - [Add peer](#add-peer)
+//! - [Get node info](#get-node-info)
+//! - [Check if consensus is enabled](#check-if-consensus-is-enabled)
+//! - [Change consensus status](#change-consensus-status)
+//! - [Node shutdown](#node-shutdown)
+//!
+//! # List Peers
+//!
+//! | Property    | Value |
+//! |-------------|-------|
+//! | Path        | `/api/system/v1/peers` |
+//! | Method      | GET   |
+//! | Query type  | - |
+//! | Return type | [`PeersInfo`] |
+//!
+//! Returns the list of incoming and outgoing connections for current node.
+//!
+//! [`PeersInfo`]: struct.PeersInfo.html
+//!
+//! ```
+//! use exonum_system_api::{private::PeersInfo, SystemApiPlugin};
+//! use exonum_testkit::{ApiKind, TestKitBuilder};
+//!
+//! # fn main() -> Result<(), failure::Error> {
+//! let mut testkit = TestKitBuilder::validator()
+//!     .with_plugin(SystemApiPlugin)
+//!     .create();
+//! let api = testkit.api();
+//! let info: PeersInfo = api.private(ApiKind::System).get("v1/peers")?;
+//! # Ok(())
+//! # }
+//! ```
+//!
+//! # Add Peer
+//!
+//! | Property    | Value |
+//! |-------------|-------|
+//! | Path        | `/api/system/v1/peers` |
+//! | Method      | POST   |
+//! | Query type  | [`ConnectInfo`] |
+//! | Return type | - |
+//!
+//! Adds a peer to the Exonum node. Node will attempt to connect to this peer.
+//!
+//! [`ConnectInfo`]: https://docs.rs/exonum-node/latest/exonum_node/struct.ConnectInfo.html
+//!
+//! ```
+//! use exonum_node::ConnectInfo;
+//! use exonum_system_api::SystemApiPlugin;
+//! use exonum_testkit::{ApiKind, TestKitBuilder};
+//!
+//! # fn main() -> Result<(), failure::Error> {
+//! # let address = "127.0.0.1:8080".to_owned();
+//! # let public_key = Default::default();
+//! // Obtaining address and public key of target node skipped...
+//! let connect_info = ConnectInfo {
+//!     address,
+//!     public_key,
+//! };
+//!
+//! let mut testkit = TestKitBuilder::validator()
+//!     .with_plugin(SystemApiPlugin)
+//!     .create();
+//! let api = testkit.api();
+//! api.private(ApiKind::System)
+//!     .query(&connect_info)
+//!     .post("v1/peers")?;
+//! # Ok(())
+//! # }
+//! ```
+//!
+//! # Get Node Info
+//!
+//! | Property    | Value |
+//! |-------------|-------|
+//! | Path        | `/api/system/v1/network` |
+//! | Method      | GET   |
+//! | Query type  | - |
+//! | Return type | [`NodeInfo`] |
+//!
+//! Obtains information about node.
```

Comment:
Nit / future work: I don't understand why this endpoint is called `network` or why it's not redundant given the user agent endpoint.

Reply:
I don't understand that either, but since the task is about documenting existing behavior, and not about changing the logic, I decided to go the straight way :)

Reply:
Yeah, sure. I'll add a task to Jira to clarify APIs.